docs/src/pages/components/dividers/VerticalDividerText.tsx

```diff
@@ -6,7 +6,8 @@ import Divider from '@material-ui/core/Divider';
 const useStyles = makeStyles((theme: Theme) => ({
   root: {
     width: '100%',
-    ...theme.typography.body2,
+    // TODO: Remove cast once migrated to emotion
+    ...(theme.typography.body2 as React.CSSProperties),
```

Comment:
Once typings are updated, we need to cast for the legacy styles utilities. Should we state this as a breaking change? Not sure if we can make it work for both 😕